pkg/cmd/pipeline/logs.go

```diff
@@ -152,7 +152,9 @@ func getAllInputs(opts *options.LogOptions) error {
 		return nil
 	}
 
-	if err := opts.Ask(options.ResourceNamePipeline, ps); err != nil {
+	if len(ps) == 1 {
+		opts.PipelineName = strings.Fields(ps[0])[0]
+	} else if err := opts.Ask(options.ResourceNamePipeline, ps); err != nil {
 		return nil
```

Comment:
Not related to your changes, but can we fix in this PR, we need to return an error in spite of `nil`

Reply:
Any chances you can capture in a issue instead?